basiclti/tsugi-util/src/test/org/tsugi/util/Base64DoubleUrlEncodeSafeTest.java

```diff
@@ -0,0 +1,84 @@
+package org.tsugi.util;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+import static org.junit.Assert.assertFalse;
+
+import org.junit.Test;
+
+import org.tsugi.util.Base64DoubleUrlEncodeSafe;
+
+/* This code was written by ChatGPT with the following prompt
+ * Please write a unit test for a Base64DoubleUrlEncodeSafe class
+ */
+
+public class Base64DoubleUrlEncodeSafeTest {
+
+	@Test
+	public void testEncodeAndDecode() {
+		String[] testStrings = {
+			"Hello, Base64DoubleUrlEncodeSafe!",
+			"Testing encoding and decoding.",
+			"12345",
+			"This is a longer string with more characters.",
+			"A tree 🌲 was here",
```

Comment:
shouldn't one of these be a really complex url esp one with an underscore?

Reply:
Good idea.